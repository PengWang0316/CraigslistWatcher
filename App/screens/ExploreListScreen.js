import React from 'react';
import { StyleSheet } from 'react-native';
import { SafeAreaView } from 'react-navigation';

import DarkGreenStatusBar from '../components/DarkGreenStatusBar';
import SearchHeadBar from '../components/SearchHeadBar';
import ItemsList from '../components/ItemsList';

const styles = StyleSheet.create({
  mainView: { backgroundColor: '#e0e0e0', flex: 1 }
});

const ExploreListScreen = props => (
  <SafeAreaView style={styles.mainView}>
    <DarkGreenStatusBar />
    <SearchHeadBar />
    <ItemsList />
  </SafeAreaView>
);
export default ExploreListScreen;
