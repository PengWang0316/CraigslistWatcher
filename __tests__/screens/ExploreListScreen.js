import React from 'react';
import renderer from 'react-test-renderer';

import ExploreListScreen from '../../App/screens/ExploreListScreen';

jest.mock('../../App/components/DarkGreenStatusBar', () => 'DarkGreenStatusBar');
jest.mock('../../App/components/SearchHeadBar', () => 'SearchHeadBar');
jest.mock('../../App/components/ItemsList', () => 'ItemsList');

describe('Test ExploreListScreen', () => {
  test('snapshot test', () => expect(renderer.create(<ExploreListScreen />).toJSON()).toMatchSnapshot());
});
