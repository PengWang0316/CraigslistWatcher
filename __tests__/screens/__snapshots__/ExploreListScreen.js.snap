// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ExploreListScreen snapshot test 1`] = `
<View
  collapsable={undefined}
  onLayout={[Function]}
  pointerEvents="box-none"
  style={
    Object {
      "backgroundColor": "#e0e0e0",
      "flex": 1,
      "paddingBottom": 0,
      "paddingLeft": 0,
      "paddingRight": 0,
      "paddingTop": 20,
    }
  }
>
  <DarkGreenStatusBar />
  <SearchHeadBar />
  <ItemsList
    navigation={undefined}
  />
</View>
`;
